platforms/r3-corda-ent/configuration/samples/network-cordaent.yaml

```diff
@@ -79,15 +79,14 @@ network:
       # Git Repo details which will be used by GitOps/Flux.
       # Do not check-in git_access_token
       gitops:
-        git_ssh: "ssh://git@github.com/<username>/blockchain-automation-framework.git"         # Gitops ssh url for flux value files 
+        git_https: "https://github.com/<username>/blockchain-automation-framework.git"         # Gitops https url for flux value files 
```

Comment:
this should be merged with the git_push_url. There is no need to have two URLs pointing to the same path.

Reply:
@sownak  Okay.. Will remove git_push_url from network.yaml and keep only git_https